### PR TITLE
Boolean expression update

### PR DESCRIPTION
**Pull request by SJ-Nosrat**
## Update - concepts.wip/boolean-expressions

- Add blurb to `config.json`,
- Add links to `links.json`,
- Add intro page,
- Add about page.

***Please let me know if the about page is correct and sufficient.***

Thanks in advance!

### CODE REVIEW

File: concepts.wip/boolean-expressions/.meta/config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for boolean-expressions concept",
-  "authors": [],
-  "contributors": []
+  "blurb": "A boolean expression is a logical statement that unambiguously evaluates to either true or false",
+  "authors": ["SJ-Nosrat"],
+  "contributors": ["SaschaMann", "cmcaine"]
```
Comment (SaschaMann) [inline concepts.wip/boolean-expressions/.meta/config.json:4]: Generally contributors are people who have either edited the files afterwards to some extent or who have done a thorough review that significantly impacted the proposed change. Neither is the case here, so you might want to leave it out :)

File: concepts.wip/boolean-expressions/about.md [hunk 2]
```diff
@@ -1,2 +1,76 @@
...
+## Expressions involving [numeric comparisons](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Numeric-Comparisons)
+
+```julia
+# true is numerically equal to 1
+julia> true ≥ 1
+true
+
+# √ can be typed as \sqrt<tab>
+julia> 120 % 11 == √100
+true
```
Comment (SaschaMann) [inline concepts.wip/boolean-expressions/about.md:14]: I kinda hate that `√100` is valid 😬

**Review by SaschaMann — APPROVED** [review]

File: concepts.wip/boolean-expressions/.meta/config.json (hunk 1 above)
Comment (SJ-Nosrat) [reply concepts.wip/boolean-expressions/.meta/config.json:4]: Should I leave the "contributors" array empty?

File: concepts.wip/boolean-expressions/about.md (hunk 2 above)
Comment (SJ-Nosrat) [reply concepts.wip/boolean-expressions/about.md:14]: Should I change the unicode character to `sqrt(100)`?

File: concepts.wip/boolean-expressions/.meta/config.json (hunk 1 above)
Comment (SaschaMann) [reply concepts.wip/boolean-expressions/.meta/config.json:4]: Yes, that seems more fitting here

File: concepts.wip/boolean-expressions/about.md (hunk 2 above)
Comment (SaschaMann) [reply concepts.wip/boolean-expressions/about.md:14]: That's up to you, I'm fine with either even if I don't like the syntax personally

Comment (SJ-Nosrat) [reply concepts.wip/boolean-expressions/about.md]: I provided the `sqrt(100)` in a comment as an alternate to the unicode expression for the square root.

**Comment by SaschaMann** [comment]
Thank you!

**Pull request merged**